### PR TITLE
Add Microsoft Azure SDKs for JavaScript, Python and .NET as projects

### PR DESCRIPTION
**Pull request by srnagar**

### CODE REVIEW

File: _data/projects/azure-sdk-for-js.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+tags:
+- javascript
+- typescript
+- microsoft
+- azure
+- cloud
+- sdk
+upforgrabs:
+  name: Up for grabs
+  link: https://github.com/Azure/azure-sdk-for-js/labels/Up%20for%20grabs
```
Comment (ritwik12) [inline _data/projects/azure-sdk-for-js.yml:13]: @srnagar There aren't any issues labeled with this tag.

File: _data/projects/azure-sdk-for-net.yml [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+tags:
+- ".NET"
+- C#
+- microsoft
+- azure
+- cloud
+- sdk
+upforgrabs:
+  name: Up for grabs
+  link: https://github.com/Azure/azure-sdk-for-net/labels/Up%20for%20grabs
```
Comment (ritwik12) [inline _data/projects/azure-sdk-for-net.yml:13]: @srnagar There aren't any issues labeled with this tag.

File: _data/projects/azure-sdk-for-python.yml [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+site: https://github.com/Azure/azure-sdk-for-python
+tags:
+- python
+- microsoft
+- azure
+- cloud
+- sdk
+upforgrabs:
+  name: Up for grabs
+  link: https://github.com/Azure/azure-sdk-for-python/labels/Up%20for%20grabs
```
Comment (ritwik12) [inline _data/projects/azure-sdk-for-python.yml:12]: @srnagar There aren't any issues labeled with this tag.

**Review by ritwik12 — CHANGES_REQUESTED** [review]

File: _data/projects/azure-sdk-for-js.yml (hunk 1 above)
Comment (srnagar) [reply _data/projects/azure-sdk-for-js.yml:13]: @ritwik12 This label was created yesterday to register with up-for-grabs. We will be tagging existing issues/creating new ones with this label.

Is it required that these labels must have issues before merging or can it be merged now and we will start tagging existing issues with this label soon?

Comment (ritwik12) [reply _data/projects/azure-sdk-for-js.yml:13]: @srnagar  I am merging but make sure to add issues with this label as without it contributors will not be able to contribute.

**Review by ritwik12 — APPROVED** [review]

**Pull request merged**